app/service/rest.py

```diff
@@ -260,6 +260,8 @@ def create_service():
     try:
         # try-catch; just in case, we don't want to error here
         ZenDeskSell().send_create_service(valid_service, user)
+        if current_app.config["FF_SALESFORCE_CONTACT"]:
+            salesforce_client.engagement_create(valid_service, user)
```

Comment:
It's great that you've been able to keep all the logic inside the salesforce client, the changes to the existing api code are so minimal 😍

Reply:
Thanks!  Having the `salesforce_client` also made it easier to group stuff compared to our initial work in the admin PR review env.